public/components/custom_panels/custom_panel_view_so.tsx

```diff
@@ -208,7 +209,10 @@ export const CustomPanelViewSO = (props: CustomPanelViewProps) => {
   };
 
   const onRename = async (newCustomPanelName: string) => {
```

Comment:
Why is `onRename` declared twice in different places?

Reply:
One is used in list view and another is details page